packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## 0.6.17+2
 
 * Adds pub topics to package metadata.
-* Updates minimum supported SDK version to Flutter 3.7/Dart 2.19.
+* Updates minimum supported SDK version to Flutter 3.7/Dart 2.19 of the example.
```

Comment:
This line can just be removed; it was only added by accident since I didn't notice that it was example-only.